src/Symfony/Component/Console/DependencyInjection/AddConsoleCommandPass.php

```diff
@@ -77,6 +77,8 @@ public function process(ContainerBuilder $container)
                 $commandName = array_shift($aliases);
             }
 
+            $isEnabled = \array_key_exists('is_enabled', $tags[0]) ? $tags[0]['is_enabled'] : true;
```

Comment:
new feature? do all commands defining default isEnabled at the code level have some issue currently?

Reply:
```suggestion
            $isEnabled = $tags[0]['is_enabled'] ?? true;
```

Reply:
```suggestion
            $isEnabled = $tags[0]['is_enabled'] ?? true;
```

Reply:
@ro0NL If you add such commands via `AddConsoleCommandPass`, it seems so. But this isn't something, @chr-hertel's changes introduced.

@chr-hertel: Can you add a small test for the new behavior of `AddConsoleCommandPass` and document the new `is_enabled` attribute in the Console component's ChangeLog?

Reply:
im wondering why this commands need this change yes :) :+1: fixing things meanwhile, but i believe `->tag('console.command', ['is_enabled' => null])` is intetional :) which is maybe bad-DX? dunno :)

Reply:
> if you add such commands via AddConsoleCommandPass

AFAIK that's any command.

Reply:
moved this to https://github.com/symfony/symfony/pull/42679